### DISCUSSION TITLE
Move to given coordinates on the Image.

### DISCUSSION THREAD

**Discussion by vivsamcom** [openseadragon/openseadragon#2274] [Q&A]
We have following requirements wherein we have to achieve following on Image Viewer. Is this feasible ? 

Clicking on Annotated Image should take the user to the same co-ordinates on the image where it was taken. 
Here while annotating image we will have to save its co-ordinates.

ROI (Region of Interest) - Same as above. 
Clicking on the ROI image (generated by AI) will take the user to original image at the same co-ordinates which is referred by ROI. 
Here AI will also give us the co-ordinates up to certain zoom level.

Can this be achieved using method  zoomTo(zoom, refPointopt, immediatelyopt) → {[OpenSeadragon.Viewport](https://openseadragon.github.io/docs/OpenSeadragon.Viewport.html)}

Which other method can be used to move to specific coordinates on the image?

Thanks in advance.

**Comment by iangilman** [comment]
You can go to specific coordinates using a combination of `zoomTo` and `panTo`. To save the values when capturing the coordinates, you can use `getZoom` and `getCenter`. 

Another approach is to use `fitBounds`; instead of a zoom and pan, you give it a rectangle in viewport coordinates. To capture the current coordinates, you can use `getBounds`. 

All of the above functions are part of the Viewport. 

Which method to use kind of depends on your use case; you might try both and see which works best. The differences show up when the viewer aspect ratio (and possibly size) are different between capturing and using the coordinates. Because fit/getBounds uses a rectangle, it necessarily includes the aspect ratio.
